Examples/create-windows-online-cloud-image.ps1

```diff
@@ -0,0 +1,37 @@
+git clone https://github.com/cloudbase/windows-openstack-imaging-tools.git
+pushd windows-openstack-imaging-tools
+Import-Module .\WinImageBuilder.psm1
```

Comment:
In orde to have the updates and MAAS jooks, please add:

`git submodule update --init`